modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
```

Comment:
This file is added in #17826. Is it meant to be included in this PR as well?

Reply:
Hi @aburdenthehand,

I think I did a save to this file after a non-significant change and then added it. Probably not the best decision. Can I just delete the file from this PR in GitHub or should I do a new PR?

Thanks

Bob

Reply:
I guess I can just do a **git checkout --** on this file and create another commit, right? :-)

Reply:
@bgaydosrh - Yeah, that'll fix it.

Reply:
Thanks, did 
$ git checkout CNV_3044 -- modules/cnv-add-nic-to-vm.adoc
and
$ git rm modules/cnv-add-nic-to-vm.adoc

and it worked like a charm.